datasets/s1-orbits.yaml

```
@@ -0,0 +1,48 @@
+Name: Sentinel-1 Precise Orbit Determination (POD) Products
+Description: >
+  Sentinel-1 Precise Orbit Determination (POD) products contain auxiliary data on satellite position/velocity for the European Space Agency's (ESA) Sentinel-1 satellite constellation and are a necessary input for nearly all Sentinel-1 data processing workflows. Sentinel-1 is a C-band Synthetic Aperture Radar (SAR) satellite constellation first launched by ESA in 2014 as part of the European Union's Copernicus Earth Observation programme.
+  
+  This dataset includes two types of POD files that are both redistributed from ESA; RESORB and POEORB. RESORB POD files are restituted orbit files generated within 180 min from sensing time and distributed daily, with accuracy requirement of 10 cm in 2D RMS, but typically below 5 cm. RESORB POD files from the last 90 days are included in this dataset. POEORB POD files are precise orbit files generated with a timeliness of 20 days from sensing time and distributed daily, with accuracy requirement of 5 cm in 3D RMS, but typically below 1 cm. Once available, these files supersede the RESORB products, as they contain the same data with better accuracy. These products are available from the entire length of the Sentinel-1 mission. The most recent POD files are added to this dataset within 20 minutes of their publication to the [Copernicus Dataspace Ecosystem](https://documentation.dataspace.copernicus.eu/Data/ComplementaryData/Additional.html#sentinel-1-orbits).
+
+  For more information on the orbit products, see the [Copernicus POD Specification](https://sentinels.copernicus.eu/documents/d/sentinel/copernicus-pod-service-file-format-specification).
```

I found this page is a more accessible introduction:

```suggestion
  For more information on the orbit products, see the [POD Sentinel-1 Products Specification](https://sentiwiki.copernicus.eu/web/s1-processing#S1-Processing-Products-Description).
```